### PR TITLE
Update capture-system-insights-from-aks.md deploy to azure integration

### PR DESCRIPTION
**Pull request by mayasingh17**
We are adding the one click functionality to deploy to Azure, Inspektor Gadget on an AKS Cluster

# Pull request guidance

Thank you for submitting your contribution to our support content! Our team works closely with subject matter experts in CSS and PMs in the product group to review all content requests to ensure technical accuracy and the best customer experience. This process can sometimes take one or more days, so we greatly appreciate your patience.

We also need your help in order to process your request as soon as possible:

- We won't act on your pull request (PR) until you type "**#sign-off**" in a new comment in your pull request (PR) to indicate that your changes are complete.

- After you sign off in your PR, the article will be tech reviewed by the PM or SME if it has more than minor changes. Once the article is approved, it will undergo a final editing pass before being merged.

### CODE REVIEW

**Comment by mayasingh17** [comment]
@blanquicet at your convenience can you please take a look at this? I didn't want to fully open the PR and notify everyone, so this is just a draft PR but I can move it to "ready for review" once you think it looks good.

File: support/azure/azure-kubernetes/capture-system-insights-from-aks.md [hunk 1]
```diff
@@ -84,6 +84,12 @@ This section outlines the steps for installing Inspektor Gadget in your AKS clus
   > [!WARNING]
   > Many mechanisms are available to deploy and use Inspektor Gadget. Each of these mechanisms is tailored to specific use cases and requirements. You can use the kubectl gadget plug-in to apply several of these mechanisms, but not all of them. For instance, deploying Inspektor Gadget by using the `kubectl gadget` plug-in depends on the availability of the Kubernetes API server. If you can’t depend on such a component because its availability might be occasionally compromised, we recommend that you avoid using the `kubectl gadget`deployment mechanism. For more information about this and other use cases, see the [Inspektor Gadget documentation](https://aka.ms/ig).
 
+### One-Click Inspektor Gadget deployment
+
+The following button automates the creation of an AKS cluster and executes the steps described in this section to deploy Inspektor Gadget in the cluster. After the deployment is complete, you can explore all of Inspektor Gadget's features in the provided shell environment.
```
Comment (blanquicet) [inline support/azure/azure-kubernetes/capture-system-insights-from-aks.md:89]:
```suggestion
The following button automates the creation of an AKS cluster and executes the steps described in this section to deploy Inspektor Gadget in a cluster. After the deployment is complete, you can explore all of Inspektor Gadget's features in the provided shell environment. 
```
?

**Review by blanquicet — APPROVED** [review]
LGTM!

**Comment by mayasingh17** [comment]
#sign-off

**Comment by blanquicet** [comment]
#sign-off

**Pull request merged**